app/assets/javascripts/projects/index.js

```diff
@@ -223,7 +225,7 @@ var ProjectsIndex = (function() {
         type: 'POST',
         dataType: 'json',
         data: {
-          project_ids: selectedProjects,
+          project_ids: selectedProjects.length > 0 ? selectedProjects : [singleSelectedProject],
```

Comment:
and what if singleSelectedProject is also empty?

Reply:
> and what if singleSelectedProject is also empty?

it will always have a value as long as export project modal is initialized